README.md

```diff
@@ -34,6 +33,57 @@ Special notes:
  * Since we cannot distribute Java Richards, a patch is required to derive the
    tracing version (`patches/trace_java_richards.diff`)
 
+## Using the scripts here to analyse an existing benchmark
+
+### Mandatory requirements
+
+  * Python 2.7 (the code here is not Python 3.x ready)
+  * bzip2 / bunzip2
+  * Python modules: numpy, rpy2
+  * R
+
+### Optional requirements
+
+  * Python modules required for plotting: matplotlib, seaborn
+  * Required for generating LaTeX tables: pdflatex
+
+### Setting up R
+
+To run the scripts here, it is necessary to install some R packages. To do this,
+open R on the command line, and run the following commands:
+
+```R
+> install.packages("devtools")
+> devtools::install_github("rkillick/changepoint")
+```
```

Comment:
Is it possible to have it install into the current directory (i.e. the warmup experiment repo) rather than dumping stuff elsewhere?